src/index.ts

```diff
@@ -165,11 +165,11 @@ function toMiddleware(this: ExpressApiRouter,
         }),
         catchError((err: Error) => {
           return resolve(formatError(err)).pipe(map((jsonError) => {
-            return new ApiResponse(jsonError || internalServerError, 500);
+            return new ApiResponse(jsonError || internalServerError, (jsonError as any).statusCode || 500);
```

Comment:
jsonError may be undefined it seems. reading `.statusCode` from undefined would fail